pkgs/applications/editors/vim/plugins/generated.nix

```diff
@@ -17829,8 +17829,8 @@ final: prev:
     meta.homepage = "https://github.com/shaunsingh/moonlight.nvim/";
   };
 
-  render-markdown = buildVimPlugin {
-    pname = "render-markdown";
+  render-markdown.nvim = buildVimPlugin {
+    pname = "render-markdown.nvim";
```

Comment:
This should be `render-markdown-nvim` as this is what the update script would do.

Also, please add an alias to deprecated.nix